Automattic/wp-calypso
# Add Payments Features Video Course

#### Proposed Changes

This PR propose add modal for new Payments Features video course. 

#### Testing Instructions

* Use [calypso live link](https://calypso.live/?image=registry.a8c.com/calypso/app:commit-d9b85e7245306cff95934e016a7504d6818e6874)
* Make sure you are not sandboxed with Public API
* In local calypso select a site and go to "My Home"
* make sure your website is lunched (see how to lunch on video below)
* after lunching find educations cards and slide to card "Make money from your website"
* click on "Start making money"
* You should see the black modal with video curse inside
* Compare texts with document from this thread: p1661852314754689-slack-CJS75TX3R

https://user-images.githubusercontent.com/82798599/185854277-f2679028-93ee-4c86-99d4-96a377ee468c.mp4



*

#### Pre-merge Checklist






Related to #65447
